stl/inc/system_error

```diff
@@ -96,7 +96,7 @@ public:
     }
 #endif // !_HAS_CXX20
 
-// TRANSITION, GH-489
+

```

Comment:
Extra empty lines don't look useful here, remove them as well

Reply:
Hi, @AlexGuteniev . Thanks for your inputs
i had removed the empty lines made a new commit to this pr.
kindly review once again.